### PR TITLE
#3 Initiazlied Mantine UI in the project!

### PR DESCRIPTION
**Pull request by mohitkumaragrawal**
Issue: #3 
Initialized Mantine UI component library in the project!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,11 @@
   },
   "dependencies": {
     "@apollo/client": "3.7.0",
+    "@emotion/react": "^11.10.4",
+    "@emotion/server": "^11.10.0",
+    "@mantine/core": "^5.6.1",
+    "@mantine/hooks": "^5.6.1",
+    "@mantine/next": "^5.6.1",
```
Comment (zaxovaiko) [inline package.json]: Could you, please, remove carets `^` from the package versions?
```suggestion
    "@emotion/react": "11.10.4",
    "@emotion/server": "11.10.0",
    "@mantine/core": "5.6.1",
    "@mantine/hooks": "5.6.1",
    "@mantine/next": "5.6.1",
```

**Review by zaxovaiko — CHANGES_REQUESTED** [review]
Just commit suggested changes and that's it 👍🏻

File: package.json (hunk 1 above)
Comment (zaxovaiko) [reply package.json]: Also, merge current `development` into your branch and solve conflicts for `package-lock.json`.

Comment (mohitkumaragrawal) [reply package.json]: Hey I fixed the issues, check it out. I think that prettier configuration is not right, eslint was not happy with the code formatting so I had to manually format the code in accordance to eslint, but that issue is out of the scope for this pull request. So, please merge it.

**Comment by mohitkumaragrawal** [comment]
@zaxoavoki Hey, please review it.

**Review by zaxovaiko — APPROVED** [review]
Looks good!

**Comment by mohitkumaragrawal** [comment]
@zaxoavoki Can you please add hacktoberfest and hacktoberfest-accepted label?

**Comment by zaxovaiko** [comment]
@mohitkumaragrawal the repo has `hactoberfest` tag by itself so you should be able to see PR on hacktoberfest page. But ok, I will add it here too 😃

**Pull request merged**